src/mazela-artur/Fsd.Artur.Cs2/Fsd.Artur.Cs2/ConsoleActions/Writers/EndMessage.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Fsd.Artur.Cs2.Animals;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Artur.Cs2.ConsoleActions.Writers
+{
+    class EndMessage
+    {
+        public void MakeNoise(List<string>[] animals )
+        {
+            for(int i=0; i<4; i++)
+            {
+                for(int y=0; y<animals[i].Count; y++)
```

Comment:
To mi się niezbyt podoba - metoda sugeruje że będziesz "przedstawiał zwierzaki" a Ty je tworzysz. Rozdziel to na 2 odpowiedzialności: fabrykę zwięrząt która dostarczy obiekt List<Animal> oraz presenter zwierząt, do którego tą listę prześlesz